a3p-integration/proposals/n:upgrade-next/package.json

```diff
@@ -31,12 +31,14 @@
     "lint-fix": "yarn lint:eslint --fix",
     "lint": "run-s --continue-on-error 'lint:*'",
     "lint:types": "tsc",
-    "lint:eslint": "eslint ."
+    "lint:eslint": "eslint .",
+    "postinstall": "patch-package"
```

Comment:
Shouldn't we use yarn4 style patches?

Reply:
No strong feelings here; this was selected to align with the [top-level package.json](https://github.com/Agoric/agoric-sdk/blob/739022455dbfb7e7d7d90e8ae7cef948cde3e97a/package.json#L71).